tests/test_models.py

```diff
@@ -37,11 +42,17 @@ def test_models_update(parser, client, metadata, preprocess_config, name_and_des
         util.main_parser(parser, client, args)
 
 
-def test_models_list(parser, client, list_defaults):
+@pytest.mark.parametrize('owner', [
+    ('--owner', service.create_organization_id()),
+    ('--owner', 'me', '--owner', service.create_organization_id()),
```

Comment:
You don't have to repeat --owner here do you? This should also work:
```suggestion
    ('--owner', 'me', service.create_organization_id()),
```